sorc/lsm_routines.fd/noah.fd/set_soilveg_snippet.f90

```diff
@@ -85,6 +93,51 @@ subroutine set_soilveg(isot,ivet, maxsmc, bb, satpsi, iret)
   
   iret = 0
 
-end subroutine set_soilveg
+end subroutine set_soilveg_noah
```

Comment:
@barlage Don't suppose there's a better way to do this (read them in from the same place the model does). Or are they very unlikely to change?